examples/hello.rs

```diff
@@ -13,7 +13,49 @@
 // limitations under the License.
 
 extern crate phosphorus;
+extern crate glutin;
+extern crate gfx;
+extern crate gfx_window_glutin;
+
+use gfx::traits::*;
+
+static HELLO_MARKUP: &'static str = include_str!("hello-markup.jade");
 
 fn main() {
-    phosphorus::say_hello();
+    // Set up our window
+    let window = glutin::WindowBuilder::new()
+        .with_vsync()
+        .with_dimensions(1280, 720)
+        .with_title(String::from("Hello"))
+        .build_strict().unwrap();
+    let mut canvas = gfx_window_glutin::init(window).into_canvas();
+
+    // Set up our phosphorus gui
+    let root = phosphorus::widgets::LayoutWidgetBuilder::new()
+        .with_background_color([0.9, 0.3, 0.3])
+        .build();
+    let mut gui = phosphorus::Gui::new(&mut canvas.factory, root);
+
+    'main: loop {
+        // Quit when the window is closed
+        for event in canvas.output.window.poll_events() {
+            match event {
+                glutin::Event::Closed => break 'main,
+                _ => {},
+            }
+        }
```

Comment:
Wondering if this is better:
```rust
if canvas.output.window.poll_events() == glutin::Event::Closed {
    break 'main;
}
```
Won't work if glutin doesn't impl Eq, but in that case an if-let would probably do the same (but that would be hacky).